website/docs/guides/delivery.mdx

```diff
@@ -0,0 +1,473 @@
+---
+title: Progressive Delivery using Flagger
+hide_title: true
+sidebar_position: 6
+---
+
+import TierLabel from "../_components/TierLabel";
+
+# Progressive Delivery using Flagger <TierLabel tiers="enterprise" />
+
+Weave GitOps Enterprise includes the ability to view [Flagger](https://docs.flagger.app/) [canary](https://docs.flagger.app/usage/how-it-works#canary-resource) objects information in the UI through the Applications > Delivery view.
+
+This guide uses Flux manifests to install Flagger and Linkerd. Using Flux allows us to manage our cluster applications in a declarative way through changes in a Git repository.
```

Comment:
- No reason, Linkerd may be installed before Flagger
- I guess it could be skipped, the most important thing is to get Prometheus to scrape the right metrics otherwise Flagger won't be able to complete the analysis. This example uses the built-in Prometheus instance that comes with Linkerd but I don't expect many people to use this in production, they will likely have other Prometheus instances already running. Generally speaking Flagger only needs the prometheus URL.

Reply:
I decided to go for a detailed guide because of the nature of Flagger: the webhooks section will need to refer to services within the cluster so having the right names/namespaces is necessary for Flagger to work correctly.

Reply:
> I decided to go for a detailed guide because of the nature of Flagger: the webhooks section will need to refer to services within the cluster and having the right names/namespaces is necessary for Flagger to work correctly.

Agree with your approach 100%

Reply:
what I would suggest is (feel free to reword as you see fit):

- expand the outline - "This guide uses Flux manifests to install Flagger and Linkerd." -> "Flagger can work with a number of service meshes and ingress controllers, to support various progressive delivery [deployment strategies](https://docs.flagger.app/usage/deployment-strategies)."

keep your bit about Using Flux..., then add:

In this guide we will walk you through a full end-to-end scenario where you will:
  - Install the Linkerd service mesh (href)
  - Install Flagger (href)
  - Deploy a sample [application]((https://github.com/stefanprodan/podinfo)) using a [canary](https://docs.flagger.app/usage/deployment-strategies#canary-release) release strategy based on metrics provided through Linkerd's in-built Prometheus instance(href)
  
Then, add a pre-requisites section which states:
- This guide assumes you already have a Kubernetes cluster running and have bootstrapped Flux. To apply the manifests listed in this guide, you will need to commit them to a repository being reconciled with Flux. For help installing Flux, you can follow their [getting started](https://fluxcd.io/docs/get-started/) documentation.


---
These changes means you could remove (at least) "In this case, Flagger is configured to work with the Linkerd service mesh. [Other providers](https://docs.flagger.app/install/flagger-install-on-kubernetes) may also be used instead of Linkerd to integrate with Flagger."